wolox-training/rof-angular
# Implemented login and logout functionalities

## Summary

Implemented login and logout functionalities

## Test Plan

* Open the project
* Try accessing login by clicking on header button, clicking on the link in the index body, or accessing a restricted view (/home)
* Create an user on register view
* Log in with your credentials
* Check you can access /home
* Check new button logout on header
* Log out 

## Trello Card 

https://trello.com/c/qLIVApmC

## Screenshot
![login](https://cloud.githubusercontent.com/assets/2911971/10592251/40d6b828-7691-11e5-84ee-b1a800836d39.png)
